_pages/index.md

```diff
@@ -17,7 +17,7 @@ layout: home
     <ol class="usa-process-list usa-prose margin-bottom-4">
       <li class="usa-process-list__item">
         <p>
-          Your integration with Login.gov starts in the <a href="https://dashboard.int.identitysandbox.gov/" class="usa-link">dashboard</a> where you can register your app.
+          Your integration with Login.gov starts in the <a href="https://developers.login.gov/testing/#using-the-sandbox" class="usa-link">dashboard</a> where you can register your app.
```

Comment:
This should use `baseurl` so the link points to the local site in local development, or the preview site in the branch preview.

```suggestion
          Your integration with Login.gov starts in the <a href="{{ site.baseurl }}/testing/#using-the-sandbox" class="usa-link">dashboard</a> where you can register your app.
```